camPod/Classes/EditPhoto/VIewModel/PhotoEditorViewModel.swift

```diff
@@ -236,7 +236,23 @@ public class PhotoEditorViewModel {
 
             view?.displayPresetCropView(rect: rect)
         case .landscape:
-            break
+            var rectWidth = isLandscape ? scaledHeight : scaledWidth
+            var rectHeight = isLandscape ? scaledHeight : scaledWidth
+
+            rectWidth! /= 3
+            rectWidth! *= 2
+            rectHeight! /= 3
+            if isLandscape {
+                rectOriginX = CGFloat(widthMiddle) - CGFloat(Int(rectWidth!/2))
+                rectOriginY = CGFloat(imageView.frame.size.height) - scaledHeight
+            } else {
+                rectOriginX = imageView.frame.origin.x + (imageView.frame.size.width - scaledWidth)/2
+                rectOriginY = heightMiddle - scaledHeight/5
+            }
```

Comment:
I would suggest maybe use a guard statement so you won't have as many forced non-null instances

Reply:
Cool, will change that, maybe it fixes the problem I have